code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -114,7 +114,7 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 		var/cached_gases = gases
 		TOTAL_MOLES(cached_gases, .)
 		. *= R_IDEAL_GAS_EQUATION * temperature / volume
-		return
+		return .
```

Comment:
Wha

Reply:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STANDARDS.md#dot-variable-

It's correct